web3/utils/abi.py

```diff
@@ -158,46 +167,179 @@ def _get_fallback_function_abi(contract_abi: ABI) -> ABIFallback:
         raise ABIFallbackNotFound("No fallback function was found in the contract ABI.")
 
 
+def _get_any_abi_signature_with_name(element_name: str, contract_abi: ABI) -> str:
+    """
+    Find an ABI identifier signature by element name. A signature identifier is
+    returned, "name(arg1Type,arg2Type,...)".
+
+    This function forces a result to be returned even if multiple are found. Depending
```

Comment:
Might be worth putting here that the first match gets returned. Is it the first match based on the ABI order?